python/lsst/daf/persistence/butler.py

```diff
@@ -136,8 +138,15 @@ def __init__(self, root, mapper=None, **mapperArgs):
         self.log = pexLog.Log(pexLog.Log.getDefaultLog(),
                 "daf.persistence.butler")
 
-    def getKeys(self, datasetType=None, level=None):
+    def registerDatasetTypeAlias(self, type, alias):
```

Comment:
Use of `type` is discouraged as it is a Python built-in.  I tend to use `datasetType` instead.